CHANGELOG.md

```diff
@@ -38,13 +38,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   - `connectCarouselFilmStrip`: Fixed `botAvatarInitials` and `userAvatarInitials` functionality from [recent name change](https://github.com/Microsoft/BotFramework-WebChat/pull/1486),
   - `BasicTranscript`: Fixed user activity should not be recreated after receive ACK from Direct Line,
   - by [@corinagum](https://github.com/corinagum) in PR [#1528](https://github.com/Microsoft/BotFramework-WebChat/pull/1528)
+- Fix [#1539], outgoing typing indicators are not sent and acknowledged properly, in PR [#1541](https://github.com/Microsoft/BotFramework-WebChat/pull/1541)
 
 ### Removed
 - `botAvatarImage` and `userAvatarImage` props, as they are moved inside `styleOptions`, in PR [#1486](https://github.com/Microsoft/BotFramework-WebChat/pull/1486)
 
 ### Samples
 - UI: [Hide upload button](https://microsoft.github.io/BotFramework-WebChat/05.d.hide-upload-button-styling/), in [#1491](https://github.com/Microsoft/BotFramework-WebChat/pull/1491)
 - UI: [Avatar image](https://microsoft.github.io/BotFramework-WebChat/04.b.display-user-bot-images-styling/), in [#1486](https://github.com/Microsoft/BotFramework-WebChat/pull/1486)
+- UI: [Send typing](https://microsoft.github.io/BotFramework-WebChat/07.b.customization-send-typing), in [#1541](https://github.com/Microsoft/BotFramework-WebChat/pull/1541)
```

Comment:
`typing` to me could mean different things and it isn't immediately clear. Because this is meant to be user facing, can the label say `Send status typing` or something that makes a bit more clear?

Reply:
how about `Send typing indicator` for channel-wide consistency?

Reply:
@cwhitten I discussed this with @compulim in person, how does `send typing indicator` sound to you? Otherwise we will stick with `Send status typing` :)

Reply:
`send typing indicator` works for me

Reply:
Renamed to "send typing indicator".

For now, I only renamed the sample code. For rename of action, reducer, sagas, props, and deprecation notes, in another PR.